### ISSUE TITLE
Fetching non-english emails cause unreadable symbols

### ISSUE THREAD

**Issue by ghost** [pvdvreede/support_helpdesk#20]
Thank you for good plugin, it's really works good but just with english mails.
If in the mail's body using non-english symbols (ex, russian, chinese, etc) it will cause unreadably symbols in issue body, ex. ������������

Checked on redmine 2.2.4

Is it possbile to fix this little, but very annoying bug ?

Thank you in advance.

**Comment by pvdvreede** [comment]
yeah, I have found this problem as well and am trying to fix it for version 2.0.0.

I am working on it now and depending on other work commitments I should have v2 ready in a month or two.

Note that version 2.0.0 will be a bit more complicated to setup and will require Redis.

**Comment by pvdvreede** [comment]
I should also add, that if I fix it for version 2 and it is a small change, I can backport the fix into v1.
